tools/runut.cmd

```diff
@@ -35,7 +35,7 @@ set _EarlyTestFail=%ERRORLEVEL%
     %*
 
 if %_EarlyTestFail% NEQ 0 (
-    exit %_EarlyTestFail%
+    exit /b %_EarlyTestFail%

```

Comment:
Oh, thank you @zadjii-msft. XD

FWIW, I am surprised that this didn't blow up my session when I ran `runut`

Reply:
were you in PowerShell?

Reply:
nope! I explicitly launched cmd with razzle.cmd to try to be you